plugins/antispam/ReadMe.md

```diff
@@ -0,0 +1,22 @@
+# Cod4X Antispam plugin
+
+#### Intelligently reduces chat spam in servers
+#
+----------------------------------------------------------------------------
+#
+#
+#
+#
+#
+#
+#
+#
+
+CVAR | DESCRIPTION
+------------- | --------------------------
+__antispam_maxMessagesPerMinute__ | *Count of maximum messages a player can send in a minute. 0 disables the chat completely. Value Range : (0 - 30)*
+__antispam_minAdminPower__ | *Minimum power points which disable the spam-control. 0 means enabled for everyone. Value Range : (0 - 100)*
+__antispam_minMessageDelay__ | *Amount of time after sending a message after which the player can chat again. 0 disables the limit. Value Range : (0 - 60)*
+__antispam_renewedMessageDelay__ | *Penalize players and add penalty time if player chats before minMessageDelay. Value Range : (0 - 1)*
+
```

Comment:
@Kingwolf123 If there was supposed to be indentation... Well... you tried ;D

Reply:
I think this must be merged and whole plugin code should be moved to a seperate repository.

Reply:
Haha. I added extra lines in between the header of MD and the table by mistake. Not big on markdown.